src/app/features/home/home.component.ts

```diff
@@ -1,12 +1,17 @@
-import { ChangeDetectionStrategy, Component } from "@angular/core";
-import { RouterOutlet } from "@angular/router";
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+import { NavbarComponent } from '../ui';
 
 @Component({
-  selector: "app-home",
+  selector: 'app-home',
   standalone: true,
-  imports: [RouterOutlet],
-  template: ` <router-outlet></router-outlet> `,
+  template: `
+    <router-outlet>
+      <app-navbar [notifications]="99"></app-navbar>
+    </router-outlet>
```

Comment:
Może przenieś navbara nad ```<router-outlet>```, bo on zawsze jest wyświetlany i zawsze jest nad resztą komponentów.
```suggestion
      <app-navbar [notifications]="99"></app-navbar>
    <router-outlet></router-outlet>    
```